backend/app/services/llm/jobs.py

```
@@ -448,38 +437,233 @@ def execute_job(
+            error="Unexpected error occurred",
+            llm_call_id=llm_call_id,
+        )
+
+
+def execute_job(
+    request_data: dict,
+    project_id: int,
+    organization_id: int,
+    job_id: str,
+    task_id: str,
+    task_instance,
```

Ruff (0.15.1)

[warning] Unused function argument: `task_instance`

(ARG001)